### ISSUE TITLE
on the main screen of the dota, there is a flaw in the form of the opening number of the 3rd act of the event

### ISSUE THREAD

**Issue by Turchik123** [ValveSoftware/Dota2-Gameplay#19751]
### Cosmetic Item Name

main page

### Description

on the main screen of the dota, there is a flaw in the form of the opening number of the 3rd act of the event 39 JULY

### Example Match ID

main page

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/174676510/44e2165c-151e-4688-adc4-7bf8c25266ed)

**Comment by Jordan11599** [comment]
Non-issue. Joke by Valve saying that they were releasing "late June"
